packages/block-editor/src/components/link-control/index.js

```diff
@@ -236,6 +236,7 @@ function LinkControl( {
 		) {
 			onChange( {
 				url: currentInputValue,
+				opensInNewTab: value?.opensInNewTab || undefined,
 				title: internalTextValue,
```

Comment:
```suggestion
				...value,
				url: currentInputValue,
				title: internalTextValue,
```

Using the spread syntax, other values can be maintained.